source/puppet/5.3/hiera_config_yaml_5.md

```diff
@@ -33,21 +33,21 @@ Global      | [`$confdir`][confdir]`/hiera.yaml`              | `/etc/puppetlabs
 Environment | [`<ENVIRONMENT>`][environment]`/hiera.yaml` | `/etc/puppetlabs/code/environments/production/hiera.yaml` `C:\ProgramData\PuppetLabs\code\environments\production\hiera.yaml`
 Module      | [`<MODULE>`][module]`/hiera.yaml`                           | `/etc/puppetlabs/code/environments/production/modules/ntp/hiera.yaml` `C:\ProgramData\PuppetLabs\code\environments\production\modules\ntp\hiera.yaml`
 
-> Note: You can change the location for the global layer's hiera.yaml with Puppet’s `hiera_config` setting.
+> Note: You can change the location for the global layer's hiera.yaml with Puppet's `hiera_config` setting.
 
-Related topics: [$confdir][confdir], [<MODULE>][module].
+Related topics: [$confdir][confdir], [`<MODULE>`][module].
 
 {:.concept}
 ## Config file syntax (hiera.yaml v5)
 
 `hiera.yaml` is a YAML file, containing a hash with up to four top-level keys:
 
-* `version` - Required. Must be the number 5, with no quotes.
-* `hierarchy` - An array of hashes, which configures the levels of the hierarchy.
-* `default_hierarchy` - An array of hashes, which sets a default hierarchy to be used only if the normal hierarchy entries do not result in a value. Only allowed in a module's hiera.yaml.
-* `defaults` - A hash, which can set a default datadir, backend, and options for hierarchy levels.
+-   `version` - Required. Must be the number 5, with no quotes.
+-   `hierarchy` - An array of hashes, which configures the levels of the hierarchy.
+-   `default_hierarchy` - An array of hashes, which sets a default hierarchy to be used only if the normal hierarchy entries do not result in a value. Only allowed in a module's hiera.yaml.
+-   `defaults` - A hash, which can set a default datadir, backend, and options for hierarchy levels.
 
-```
+``` yaml
```

Comment:
Adding a language indicator to the code block helps communicate how the block's syntax highlighting should work. This is less necessary since we moved to Drupal, but can still be helpful context.